shared/lib/i18n-utils/utils.js

```diff
@@ -1,16 +1,24 @@
 /**
  * External dependencies
  */
-var find = require( 'lodash/collection/find' );
+import find from 'lodash/collection/find';
 
 /**
  * Internal dependencies
  */
-var config = require( 'config' );
+import config from 'config';
 
-var i18nUtils = {
+const localeRegex = /^[A-Z]{2}$/i;
+const localeWithCountryRegex = /^[A-Z]{2}-[A-Z]{2}$/i;
```

Comment:
Yeah, I wasn't sure which was more appropriate. `region` sounds good to me. :+1: